packages/test/test-end-to-end-tests/src/test/mockDetachedBlobStorage.ts

```diff
@@ -33,10 +35,25 @@ export class MockDetachedBlobStorage implements IDetachedBlobStorage {
 }
 
```

Comment:
maybe put the `driversThatSupportBlobs` array here and reuse it.

Reply:
it could also be good to just export a function, `supportsBlobs(driver: ITestDriver)` as we may need to take endpoint into account as well at some point, then all the callers would call that function, rather than using the array directly

Reply:
this will make any later changes to the support matrix a small change, rather than having to update it everywhere like we are here.

Reply:
it should be a free function, `export function driverSupportsBlobs(driver: ITestDriver)` not on any class. and it should be used to replace all the existing checks.

Reply:
the goal is that if a new server starts supporting blobs we only have one place to update